docs.html

```diff
@@ -49,94 +49,215 @@ <h1>AVATAR</h1>
+                        <div class="line">
+                            <span class="comment">/* Danger Alert */</span>
+                        </div>
+                        <div class="line">
+                            <span class="sym"><</span>
+                            <span class="element">div </span>
+                            <span class="attribute">class</span>
+                            <span class="sym">=</span>
+                            <span class="attr-content">"alert alert-danger"</span>
+                            <span class="sym">></span>
+                            <span class="element-content">Danger Alert</span>
+                            <span class="sym"><</span>
+                            <span class="sym">/</span>
+                            <span class="element">div</span>
+                            <span class="sym">></span>
+                        </div>
+                        <div class="line">
+                            <span class="comment">/* Warning Alert with an icon */</span>
+                        </div>
+                        <div class="line">
+                            <span class="sym"><</span>
+                            <span class="element">div </span>
+                            <span class="attribute">class</span>
+                            <span class="sym">=</span>
+                            <span class="attr-content">"alert alert-warning"</span>
+                            <span class="sym">></span>
+                            <span class="sym"><</span>
+                            <span class="element">i </span>
+                            <span class="attribute">class</span>
+                            <span class="sym">=</span>
+                            <span class="attr-content">"fas fa-exclamation-triangle"</span>
+                            <span class="sym">></span>
+                            <span class="sym"><</span>
+                            <span class="sym">/</span>
+                            <span class="element">i</span>
+                            <span class="sym">></span>
+                            <span class="element-content"> Warning Alert</span>
+                            <span class="sym"><</span>
+                            <span class="sym">/</span>
+                            <span class="element">div</span>
+                            <span class="sym">></span>
+                        </div>
+                        <div class="line">
+                            <span class="comment">/* Success Alert */</span>
+                        </div>
+                        <div class="line">
+                            <span class="sym"><</span>
+                            <span class="element">div </span>
+                            <span class="attribute">class</span>
+                            <span class="sym">=</span>
+                            <span class="attr-content">"alert alert-success"</span>
```

Comment:
Use descriptive class names as it improves readability and understanding.

Reply:
Okay thanks partha